Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp

```diff
@@ -585,7 +597,7 @@ void Scene_points_with_normal_item::drawPoints(CGAL::Three::Viewer_interface* vi
 
     GLfloat point_size;
     viewer->glGetFloatv(GL_POINT_SIZE, &point_size);
-    viewer->glPointSize(4.f);
+    viewer->glPointSize(d->point_Slider->value());
```

Comment:
The size in glPointSize is in pixels, not in World coordinates, so it is not related to the Bbox.